src/ReactiveUI.Tests/ObservableAsPropertyHelperTest.cs

```diff
@@ -248,6 +226,47 @@ public void ToPropertyShouldFireBothChangingAndChanged()
         }
 
         [Fact]
+        public void ToProperty_NameOf_ShouldFireBothChangingAndChanged()
+        {
+            var fixture = new OaphNameOfTestFixture();
+
+            // NB: This is a hack to connect up the OAPH
```

Comment:
Confused by this. OAPH should be subscribing and connected up by default, unless the subscription has been defered.

Reply:
This is a copy paste from the other test nearby. If not needed, likely not needed in previously written test.